pages/common/hg.md

```diff
@@ -0,0 +1,19 @@
+# hg
+
+> Mercurial distributed source control management.
+
+- Check the Mercurial version:
+
+`hg --version`
+
+- Call general help:
+
+`hg help`
+
+- Call help on a command:
+
+`hg help {{command}}`
```

Comment:
I assume that there are/will be separate pages for each command like we have with `git`? Maybe updating the description with s see also might help there.

Reply:
Yes, I was going to do separate pages because there are a lot of commands.

A 'see also' would be good, but as I haven't yet added any additional commands would it be required yet?

Reply:
Also, should additional commands (i.e. `hg-add`, `hg-clone`, etc.) be added as separate PRs?

Reply:
Ah, I see your point! 😛 

Yeah, let's leave it until you've actually created those pages lol

Reply:
Just keep in mind to update the page whenever you submit a PR for a hg subcommand.